src/docsfy/main.py

```
@@ -54,6 +55,14 @@ async def lifespan(app: FastAPI) -> AsyncIterator[None]:
 
     _generating.clear()
     await init_db(data_dir=settings.data_dir)
+
+    try:
+        await pricing_cache.load()
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)